src/coreclr/vm/classlayoutinfo.cpp

```diff
@@ -55,9 +55,14 @@ namespace
                 pfwalk->m_placement.m_offset = ulOffset;
                 pfwalk->m_sequence = (ULONG)-1;
 
+#if FEATURE_UNITY_EMBEDDING_INTERFACE
```

Comment:
This bug may have been fixed: See https://github.com/dotnet/runtime/issues/53542 & https://github.com/dotnet/runtime/pull/54235

Reply:
Nice. Thanks for finding this @scott-ferguson-unity  !

Reply:
I tried removing this and the issue seemed to remain.